music21/stream/makeNotation.py

```diff
@@ -1486,6 +1488,74 @@ def getTiePitchSet(prior):
             tiePitchSet.add(n.pitch.nameWithOctave)
         return tiePitchSet
 
+def makeAccidentalsInMeasureStream(s: 'music21.stream.Stream', **subroutineKeywords):
+    '''
+    Makes accidentals in place on a stream consisting of only Measures
+    Helper for Stream.makeNotation and Part.makeAccidentals.
+    '''
+    if s.getElementsNotOfClass('Measure'):
+        raise ValueError(f'{s} must contain only Measures')
+
+    srkCopy = subroutineKeywords.copy()
+    for illegalKeyword in (
+        'meterStream',
+        'refStreamOrTimeRange',
+        'pitchPastMeasure',
+        'useKeySignature',
+        'searchKeySignatureByContext',
+        'tiePitchSet',
+        'inPlace',
+    ):
+        try:
+            del(srkCopy[illegalKeyword])
+        except KeyError:
+            pass
```

Comment:
Next step is just to get the four or so legal keywords left and just put them in the signature.